.github/lighthouserc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ci": {
+    "preset": "lighthouse:recommended",
+    "assert": {
+      "assertions": {
+        "categories:performance": ["error", {"minScore": 0.4}],
```

Comment:
What is this performance measuring? 
How easy would it be to define our own latency metric and plumb it through to light house (from a cursory look at their docs i remember getting the impression that it was possible - is that right)?

Reply:
It's measuring:
- First Contentful Paint (FCP)
- Speed Index
- Time To Interactive
- First Meaningful Paint
- First CPU Idle
- Max Potential First Input Delay

You can see the results in detail in the CI artefact (like [this one](https://github.com/misterpeddy/hands-down/pull/47/checks?check_run_id=569196345)) which can be downloaded when clicking on "Details" of the _"Lighthouse / static-dist-dir (pull_request)"_ check.

It's fairly straight forward to define assertions for specific metrics (e.g. FCP) as shown on: https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/assertions.md#audits.
As for custom metrics, I'm not sure.

Reply:
@misterpeddy Do you want to have more specific performance assertions?